docs/release-notes/release-notes-0.18.0.md

```diff
@@ -1,4 +1,5 @@
 # Release Notes
+- [Release Notes](#release-notes)
```

Comment:
This looks weird to me. Isn't it customary for a table of contents to only reference headings that come after it?

Reply:
It is weird, my IDE does that for some reason.
Fixed it.